assets/component-image-with-text.css

```diff
@@ -74,7 +74,7 @@
   fill: currentColor;
 }
 
-.image-with-text__media--placeholder.animate--ambient > svg {
+.image-with-text__media--placeholder:is(.animate--ambient, .animate--zoom-in) > svg {
   top: 0;
```

Comment:
Asking question so I can enhance my learning: why did you use :is() vs. :where() ?

Reply:
The main difference is in specificity.

`:is()`  will use the selectors within it to count towards the specificity of the overall selector.
`:where()` will default to a specificity of `0`. 

So in our case, because the selector used to be `.class.anotherClass` then it would have a specificity count of `2`. By using `:is()` I make sure I keep that same specificity. 

If I had used `:where()` then the specificity would be lower. 

<details>
<summary>[Here](https://specificity.keegan.st/) is a specificity calculator and its result: </summary>

![](https://screenshot.click/28-52-ixrhd-7tiho.png)
</details>

[Here is a useful breakdown](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) 🙂

Reply:
Oh wow! This is a very helpful diagram! Thank you so much for taking the time to explain the difference to me !